### PR TITLE
update the documentation of ProteinPaint-related tools

### PR DESCRIPTION
**Pull request by siosonel**

### CODE REVIEW

**Review by karishma-gangwani — APPROVED** [review]

File: docs/Data_Portal/Users_Guide/oncomatrix.md [hunk 1]
```diff
@@ -180,164 +174,170 @@ The row containing TP53 cases now moves back up in position 1 above ATRX.
...
 
-Click 'Age at diagnosis' from the options. The matrix reloads to show the following view.
+Search for the term ‘Age at diagnosis’ and Click the term. The matrix reloads to show the following view.
 
 [![Select a grouping variable](./images/oncomatrix/22-group-age-at-diag.png)](./images/oncomatrix/22-group-age-at-diag.png 'Click to see the full image.')
 
 As shown above, labels for different age groups show up vertically and all cases get distributed with a clearcut separation according to the age bins.
 
-Click on the 'Age at diagnosis' (blue pill, as shown). This opens a short menu with action items. Click on the first item 'Edit' as shown.
+Click on the ‘Age at diagnosis’ (blue pill, as shown). This opens a short menu with action items. Click on the first item ‘Edit’ as shown.
```
Comment (bullenc) [inline docs/Data_Portal/Users_Guide/oncomatrix.md:258]: Is the replacement of straight quotation marks with curved quotation marks intentional? Sometimes our doc site scripts cannot render curved quotation marks correctly and we have had to be correct these back before. See this line and lines below.

File: docs/Data_Portal/Users_Guide/oncomatrix.md [hunk 2]
```diff
@@ -180,164 +174,170 @@ The row containing TP53 cases now moves back up in position 1 above ATRX.
...
 
 [![Click the variable button to edit](./images/oncomatrix/23-edit-groups.png)](./images/oncomatrix/23-edit-groups.png 'Click to see the full image.')
 
-Drag the red lines on the density distribution to select binning or input numbers for custom binning and select 'Apply'.
+Drag the red lines on the density distribution to select binning or input numbers for custom binning and select ‘Apply’.
```
Comment (bullenc) [inline docs/Data_Portal/Users_Guide/oncomatrix.md:262]: See comment above re: curved quotation marks

File: docs/Data_Portal/Users_Guide/oncomatrix.md [hunk 3]
```diff
@@ -180,164 +174,170 @@ The row containing TP53 cases now moves back up in position 1 above ATRX.
...
 
-[![Edit the binning of values](./images/oncomatrix/24-binning.png)](./images/oncomatrix/24-binning.png 'Click to see the full image.')
+[![Click the variable button to edit bins](./images/oncomatrix/24-binning.png)](./images/oncomatrix/24-binning.png 'Click to see the full image.')
 
-The matrix reloads with new bin groupings The labels for the groups are user controlled and hence can be modified according to user requirements.
+The matrix reloads with new bin groupings The labels for the groups are user controlled and hence can be modified according to user requirements. 
 
-Click on the blue pill for 'Age at diagnosis' again and click 'Replace'. Select 'Primary site' as shown.
+Click on the blue pill for ‘Age at diagnosis’ again and click ‘Replace’. Select ‘Primary site’ as shown.
```
Comment (bullenc) [inline docs/Data_Portal/Users_Guide/oncomatrix.md:268]: See comment above re: curved quotation marks

File: docs/Data_Portal/Users_Guide/oncomatrix.md [hunk 4]
```diff
@@ -446,9 +439,45 @@ Click `Submit` and the matrix will update to reflect the selected MSigDB gene se
...
+[![CNV data](./images/oncomatrix/CNV-data.png)](./images/oncomatrix/CNV-data.png 'Click to see the full image.')
+
+To select specific CNVs, click on the 'Show selected CNV' tab in the control panel and select the CNVs you want to visualize.
+
+[![Select CNV](./images/oncomatrix/select-CNV.png)](./images/oncomatrix/select-CNV.png 'Click to see the full image.')
+
 ## Variables
 
-The third button from the left called `Variables` allows user to add in additional variables in the form of rows on the matrix. Click `Variables` to display a tree of variables and select `Disease type`, `Index date` and `Primary site`. Click the button `Submit 3 variables` as shown.
+The third button from the left called ``Variables` allows user to add in additional variables in the form of rows on the matrix. Click `Variables` to display a tree of variables and select `Disease type`, `Index date` and `Primary site`. Click the button `Submit 3 variables` as shown.
```
Comment (bullenc) [inline docs/Data_Portal/Users_Guide/oncomatrix.md:480]: Extra grave accent character in front of Variables

File: docs/Data_Portal/Users_Guide/oncomatrix.md [hunk 5]
```diff
@@ -180,164 +174,170 @@ The row containing TP53 cases now moves back up in position 1 above ATRX.
...
 
-Click on the blue pill for 'Age at diagnosis' again and click 'Replace'. Select 'Primary site' as shown.
+Click on the blue pill for ‘Age at diagnosis’ again and click ‘Replace’. Select ‘Primary site’ as shown.
 
 [![Replace the grouping variable](./images/oncomatrix/25-replace-group.png)](./images/oncomatrix/25-replace-group.png 'Click to see the full image.')
 
 The matrix reloads with the new variable distribution.
 
-The last option on the menu is 'Remove'. Click on the '917 Cases' button, followed by 'Age at diagnosis' shown in blue to reveal the menu option. Click 'Remove' to completely get rid of any groups.
+The last option on the menu is ‘Remove’. Click on the ‘917 Cases’ button, followed by ‘Age at diagnosis’ shown in blue to reveal the menu option. Click ‘Remove’ to completely get rid of any groups.
```
Comment (bullenc) [inline docs/Data_Portal/Users_Guide/oncomatrix.md:274]: See comment above re: curved quotation marks

File: docs/Data_Portal/Users_Guide/oncomatrix.md [hunk 6]
```diff
@@ -180,164 +174,170 @@ The row containing TP53 cases now moves back up in position 1 above ATRX.
...
 
 [![Remove the grouping variable](./images/oncomatrix/26-remove-group.png)](./images/oncomatrix/26-remove-group.png 'Click to see the full image.')
 
 This will remove all and any groupings and show the default view again.
 
-### Sort Case Groups
+### Adding Survival term as a grouping variable
 
-Add the variable 'Age at diagnosis' again using the 'Group Cases by' button as shown in the previous section. By default, groups are loaded ordered by their name. Change the selection to 'Case count' as shown below.
+Click on the ‘Age at diagnosis’ (blue pill, as shown). This opens a short menu with action items. Click on the second item ‘Replace’ as shown.
```
Comment (bullenc) [inline docs/Data_Portal/Users_Guide/oncomatrix.md:282]: See comment above re: curved quotation marks

File: docs/Data_Portal/Users_Guide/oncomatrix.md (hunk 1 above)
Comment (siosonel) [reply docs/Data_Portal/Users_Guide/oncomatrix.md:258]: Thanks, I'll revert to double-quotes.

Comment (siosonel) [reply docs/Data_Portal/Users_Guide/oncomatrix.md:258]: @bullenc Actually, before I'd like to verify first: by common standard, back ticks will format the enclosed text as code snippets, with a grayed out text background. I'm hesitant to remove that styling, unless it really breaks the build.

Comment (bullenc) [reply docs/Data_Portal/Users_Guide/oncomatrix.md:258]: @siosonel we can move forward and I will let you know if it breaks the build. My understanding is that the code back ticks tend to point in the other direction (``) whereas these seem to be curved quotations (‘’) vs. the original straight quotations ('').

Comment (bullenc) [reply docs/Data_Portal/Users_Guide/oncomatrix.md:258]: ah replied late, no worries, thanks!

**Review by bullenc — APPROVED** [review]

**Pull request merged**